deploy/bindata_generated.go

```diff
@@ -103,7 +103,7 @@ func deployKubernetes117DirectPmemCsiYaml() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "deploy/kubernetes-1.17/direct/pmem-csi.yaml", size: 13025, mode: os.FileMode(420), modTime: time.Unix(1619608498, 0)}
+	info := bindataFileInfo{name: "deploy/kubernetes-1.17/direct/pmem-csi.yaml", size: 13025, mode: os.FileMode(436), modTime: time.Unix(1622056533, 0)}
```

Comment:
Only the time stamps changed. You can remove the entire bindata_generated.go change from the PR.